mlflow/pipelines/steps/ingest/__init__.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, step_config: Dict[str, Any], pipeline_root: str):
         super().__init__(step_config, pipeline_root)
 
         dataset_format = step_config.get("format")
+        self.disable_profiling = step_config.get("disable_profiling", False)
```

Comment:
I think we want the this config to be something like:
```
steps:
  split:
      step_card:
          disable_profiling: true
```
So we can add more API to `step_card`. What do you think?

Reply:
Do we know exactly what we want to add to `step_card`? If not I think that adds a bit of undue complexity to this PR.

Reply:
```
steps:
  split:
     disable_profiling: true
```
is probably fine for now. We might need to make some breaking changes when we have step and step card API figured out.